gems/sorbet-runtime/lib/types/props/decorator.rb

```diff
@@ -571,12 +571,6 @@ def prop_defined(name, cls, rules={})
 
   # From T::Props::Utils.deep_clone_object, plus String
   TYPES_NOT_NEEDING_CLONE = [TrueClass, FalseClass, NilClass, Symbol, String, Numeric]
-  if defined?(Opus) && defined?(Opus::Enum)
-    TYPES_NOT_NEEDING_CLONE << Opus::Enum
-  end
-  if defined?(T::Enum)
-    TYPES_NOT_NEEDING_CLONE << T::Enum
-  end
```

Comment:
As far as I can tell, this code was a no-op before, because sorbet-runtime would always run before `lib/enum/` loaded. So point 1: it worked anyways. Point 2: if we delete this, the only thing that happens is somewhere down the line, lib/props/ tries opts to call `deep_clone_object(x)` instead of just `x.clone`. But `deep_clone_object` for a T::Enum is defined to just not do anything, because all enum values are singleton instances.